### PR TITLE
[Translation zh-cn] Shared > Udacity > ud892.html

### PR DESCRIPTION
**Pull request by limhenry**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-CN/review/287)

### CODE REVIEW

**Comment by limhenry** [comment]
The translation is based on Udacity (Chinese) website
https://cn.udacity.com/course/web-tooling-automation--ud892

**Comment by sotayamashita** [comment]
We have signed Corporate CLA but it will takes time to accept a few days according FAQ for Google CLA. https://cla.developers.google.com/about

File: src/content/zh-cn/_shared/udacity/ud892.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
```
Comment (petele) [inline src/content/zh-cn/_shared/udacity/ud892.html]: original does not contain the doctype, `html` or `body` tag. These files are included in to other files and should not include them.

**Comment by petele** [comment]
@sotayamashita & @chikathreesix - would you please take a look!

**Comment by sotayamashita** [comment]
@limhenry Do you know why feature branch is `unknown repository`?
screen shot 2017-12-08 at 16 28 02

**Comment by limhenry** [comment]
I think I accidentally deleted the fork. Em what should I do now?

**Pull request closed without merging**